cmd/notary/main.go

```diff
@@ -11,20 +10,25 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 
+	"github.com/docker/docker/pkg/homedir"
 	"github.com/docker/notary/pkg/passphrase"
 	"github.com/docker/notary/version"
 )
 
-const configFileName string = "config"
-const defaultTrustDir string = ".notary/"
-const defaultServerURL = "https://notary-server:4443"
-const idSize = 64
+const (
+	configFileName   string = "config"
+	defaultTrustDir  string = ".notary/"
+	defaultServerURL        = "https://notary.docker.io"
```

Comment:
maybe you should not have a default when you ship because otherwise you are going to be getting a whole lot of tickets with the title 
"cannot connect to http://notary-server:4443"